osu.Game/Rulesets/Mods/ModWindDown.cs

```diff
@@ -45,5 +45,14 @@ public class ModWindDown : ModTimeRamp
         };
 
         public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(ModWindUp)).ToArray();
+
+        public ModWindDown()
+        {
+            InitialRate.BindValueChanged(val =>
+                FinalRate.Value = Math.Min(FinalRate.Value, val.NewValue - 0.01));
```

Comment:
Use `Precision` directly here instead of `0.01` as a literal.

Also aren't these backwards? They sure do *read* backwards (why is the user-supplied value being decremented, instead of the bound?) Shouldn't these be
```suggestion
                FinalRate.Value = Math.Min(FinalRate.Value - FinalRate.Precision, val.NewValue));
```

(etc etc everywhere else)

Reply:
The intention is to have it so that if `InitialRate` is dragged to a value greater then `FinalRate`, the `FinalRate` is adjusted to be above that new value. To me it felt like better UX then preventing the user from dragging past that rate (since people will adjust the settings top to bottom). The above code achieves that behaviour but I agree it maybe doesnt read as well as it could.

Will make the Precision change.